pkg/metrics/kubemetrics.go

```diff
@@ -108,6 +108,9 @@ func InitKubeMetrics(clusterCache clustercache.ClusterCache, metricsConfig *Metr
 				metricsConfig:    *metricsConfig,
 			})
 		} else if opts.EmitKubeStateMetricsV1Only {
+			// We still need the kubecost_pv_info metric to look up storageclass on legacy clusters.
+			forceDisabled := []string{"kube_persistentvolume_capacity_bytes", "kube_persistentvolume_status_phase"}
```

Comment:
the v1Only is for KSM v1 metrics is to emit the metrics delta between v1 and v2 iirc. this preserves the old logical behavior since when flag was set the whole set of PV metrics would not be emitted previously. Was that behavior a bug before?